indico/modules/events/registration/client/js/form/fields/DateInput.jsx

```diff
@@ -88,10 +91,18 @@ DateInputComponent.propTypes = {
 };
 
 DateInputComponent.defaultProps = {
+  id: undefined,
```

Comment:
Intentional or missed when changing the ids to required?

Reply:
Well, do you want to require those ids? If so, then we can get rid of them.

Reply:
I mean if an id prop is always passed,I think it makes sense to mark them as required.

At least I assume that a prop which is not required isn't always provided...